<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workerMenu</class>
 <widget class="QDialog" name="workerMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="welcomeMessageLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>169</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pruductsRedactorButton">
      <property name="text">
       <string>Просмотр базы товаров</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registerButton">
      <property name="text">
       <string>Регистрация нового работника</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="activeOrdersButton">
      <property name="text">
       <string>Просмотр активных заказов</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>workerMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
